CalmspaceUITests/CalmspaceUITests.swift

```diff
@@ -58,6 +65,14 @@ final class CalmspaceUITests: XCTestCase {
             XCTFail("Web view not found")
         }
     }
+    
+    func testFullNavigationFlow() throws {
```

Comment:
This is an interesting approach, although let's write more than one test. Three or five would be great.